openshift.spec

```diff
@@ -275,7 +370,97 @@ fi
 %defattr(-,root,root,-)
 %{_bindir}/pod
 
+# ===
+%files -n atomic-enterprise
+%defattr(-,root,root,-)
+%doc README.md LICENSE
+%{_bindir}/openshift
+%{_bindir}/atomic-enterprise
+%{_bindir}/oc
+%{_bindir}/oadm
+%{_bindir}/kubectl
+%{_sharedstatedir}/origin
+/etc/bash_completion.d/*
+
+%files -n atomic-enterprise-master
+%defattr(-,root,root,-)
+%{_unitdir}/atomic-enterprise-master.service
+%config(noreplace) %{_sysconfdir}/sysconfig/atomic-enterprise-master
+%config(noreplace) /etc/%{name}/master
```

Comment:
Originally no, but now there is an email thread going on about changing both to be /etc/origin/.